stack-nightly.yaml

```diff
@@ -1 +1,8 @@
-resolver: nightly-2023-09-09
+resolver: nightly-2023-12-26
+extra-deps:
+  - crypton-1.0.0
+  - amazonka-2.0
+  - amazonka-core-2.0
+  - amazonka-sso-2.0
+  - amazonka-sts-2.0
+  - amazonka-s3-2.0
```

Comment:
Done, but I don't think CI respects the resolver version listed in the `stack-nightly.yaml` file. I'm gonna try changing the nightly file name to include that date so it reads like a nightly resolver version. If this isn't something you want, or if you'd prefer to have both a `stack-nightly.yaml` and `stack-nightly-yyyy-mm-dd.yaml`, LMK!

Reply:
It's not possible to put `resolver: nightly` in a stack-yaml file, you can only do that via `--resolver nightly`. For this reason, our `stack-action` will always add `--resolver nightly` for you, if it sees you're using a `stack-nightly.yaml`.

Therefore, on CI, the `resolver` field is ignored. So we just use it to record the data it was last worked on, since you'd want to do that and run it locally anyway to get the `.lock` file correctly updated.

As for the failure: that happens a lot, and is why the `test (stack-nightly.yaml)` check is not a Required Status. Feel free to work on it (PR to amazonka, forked dependency, etc) or you can leave it broken for now.

Reply:
That makes sense!